### PR TITLE
PC-10566 Terraform nobl9 cloudwatch redshift direct resource should return external_id

### PR DESCRIPTION
**Pull request by kubaceg**

### CODE REVIEW

File: nobl9/data_source.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/nobl9/nobl9-go/manifest/v1alpha"
+)
+
+func dataSourceAWSIAMRoleAuthExternalID() *schema.Resource {
+	return &schema.Resource{
+		Description: "[Cross account IAM roles](https://docs.nobl9.com/Sources/Amazon_CloudWatch/#cross-account-iam-roles-new)",
```
Comment (triluch) [inline nobl9/data_source.go:13]: Just having link is kinda ugly... When reading our terraform docs it makes it annoying to open 353265423 links to get anything about particular resource or data source. At least some "This returns external ID and AWS account ID that can be used to create cross-account IAM roles in AWS." would be nice.

Comment (kubaceg) [reply nobl9/data_source.go:13]: Good point :)

**Comment by klaudia-nobl9** [comment]
Tested on dev-sapphire env with `terraform state` - `account_id` and `external_id` is returned by Terraform ✅

File: docs/data-sources/aws_iam_role_external_id.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+# nobl9_aws_iam_role_external_id (Data Source)
+
+Returns external ID and AWS account ID that can be used to create [cross-account IAM roles in AWS](https://docs.nobl9.com/Sources/Amazon_CloudWatch/#cross-account-iam-roles-new).
+
+## Example Usage
+
+```terraform
+data "nobl9_aws_iam_role_external_id" "this" {
+  name = "my-slo-name"
```
Comment (triluch) [inline docs/data-sources/aws_iam_role_external_id.md:17]: Shouldn't that be `my-datasource-name` (same thing in example .tf)?

**Review by nieomylnieja — APPROVED** [review]

**Pull request merged**